DotNet/.gitignore

```diff
@@ -37,4 +37,5 @@ _ReSharper*/
 packages/
 output/
 .jsbeautifyrc
-*.orig
\ No newline at end of file
+*.orig
+CompanyLogo.cs
```

Comment:
How does this exclude auth_config.json from Git?

Reply:
I have made the necessary change to the  web/.gitignore, and it is now working as required.